peeps/PEEP-006.md

```diff
@@ -0,0 +1,30 @@
+# PEEP-006: Change the behavior of `-d` flag when generating requirement.txt
+
+Make the behavior of `pipenv lock -r -d` consistent with those in other commands: convert all dependencies.
+
+☤
+
+If you type `pipenv lock --help` the help document says:
+
+```bash
+-d, --dev           Install both develop and default packages.  [env var:PIPENV_DEV]
+```
+
+That is not accurate and confusing for `pipenv lock -r`, which only produces the develop requirments.
+
+This PEEP proposes to change the behavior of `pipenv lock -r -d` to produce **all** requirements, both develop
+and default. Also, change the help string of `-d/--dev` to **"Generate both develop and default requirements"**.
+
+Introduce a new flag `--only` to restrict to develop requirements only. The flag does nothing when not combined with
```

Comment:
In implementing this, I found it was cleaner to make the new flag `--dev-only`, so it could stand alone, and not be dependeny on also supplying `--dev` (instead, supplying both `--dev` and `--dev-only` has the same effect as just supplying `--dev-only`).